tests/test_prompts.py

```diff
@@ -0,0 +1,221 @@
+"""Tests for prompts module."""
+
+import pathlib
+import tempfile
+import unittest
+
+import pydantic
+
+from imbi_automations import models, prompts
+
+
+class PromptsTestBase(unittest.TestCase):
+    """Base test class with shared fixtures for prompts tests."""
+
+    def setUp(self) -> None:
+        """Set up test fixtures."""
+        self.temp_dir = tempfile.TemporaryDirectory()
```

Comment:
Use the context manager version and [TestCase.enterContext](https://docs.python.org/3/library/unittest.html#unittest.TestCase.enterContext) to automatically invoke the cleanup in all cases. In this case, `tearDown` will not be called if something in the `setUp` chain fails. Contexts, however, are cleaned up correctly.

```suggestion
        self.temp_dir_name = self.enterContext(tempfile.TemporaryDirectory())
```